modules/services/wlsunset.nix

```diff
@@ -63,6 +63,39 @@ in {
       '';
     };
 
+    output = mkOption {
+      type = types.str;
+      description = ''
+        Name of output to use, by default all outputs are used.
+      '';
+    };
+
+    time = {
+      duration = mkOption {
+        type = types.int;
+        description = ''
+          The duration in seconds.
+          e.g 1800
```

Comment:
Use `example = 1800` rather than adding an example in the description.

Reply:
I'll definitely do that, I forgot that existed. Thank you for pointing that out